atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/knowledge/KnownConcludedTransactionsStoreConcurrencyTest.java

```diff
@@ -86,6 +86,31 @@ public void batchesReadsUnderHighConcurrency() throws InterruptedException {
                 .get(eq(TransactionConstants.KNOWN_CONCLUDED_TRANSACTIONS_TABLE), anyMap());
     }
 
+    @Test
+    public void writesPreserveCorrectnessUnderHighConcurrencyAndMinimumTimestampSet() throws InterruptedException {
+        long minimumConcludableTimestamp = 10L;
+        int numThreads = 300;
+        List<Range<Long>> candidateTimestampRanges = LongStream.range(1, numThreads)
+                .mapToObj(index -> Range.closed(2 * index, 2 * index + 1))
+                .filter(range -> !range.isConnected(Range.closed(Long.MIN_VALUE, minimumConcludableTimestamp)))
+                .collect(Collectors.toList());
+
+        knownConcludedTransactionsStore.setMinimumConcludableTimestamp(10L);
```

Comment:
nit: let's use the variable

Reply:
lol and the funny thing that's exactly why I created it!